pkg/querier/fanout/fanout.go

```diff
@@ -0,0 +1,147 @@
+package fanout
+
+import (
+	"context"
+
+	"github.com/grafana/dskit/concurrency"
+
+	"github.com/grafana/loki/pkg/iter"
+	"github.com/grafana/loki/pkg/loghttp"
+	"github.com/grafana/loki/pkg/logproto"
+	"github.com/grafana/loki/pkg/logql"
+	"github.com/grafana/loki/pkg/querier"
+	"github.com/grafana/loki/pkg/storage/stores/index/stats"
+)
+
+const readConcurrency = 5
```

Comment:
This should be configurable :slightly_smiling_face:

Reply:
thanks,done ,add queirer.MaxRemoteReadConcurrent config .